tests/integration_tests/test_cli.py

```diff
@@ -1268,7 +1268,259 @@ def mock_get_wallet(*args, **kwargs):
                 address=wallet.coldkeypub.ss58_address
             )
             self.assertAlmostEqual(balance.tao, mock_balance.tao, places=4)
-                
+        
+    def test_nominate( self ):
+        config = self.config
+        config.command = "nominate"
+        config.no_prompt = True 
+        config.wallet.name = "w0"
+        config.hotkey = "hk0"
```

Comment:
lets also add `subtensor.network= finney`, just incase there are many networks in the future

Reply:
Did `subtensor.network = "finney"` and `subtensor._mock = True`